conans/test/unittests/util/tools_test.py

```diff
@@ -785,6 +785,25 @@ def get(self, *args, **kwargs):
                        requester=requests, out=out)
         http_server.stop()
 
+    @attr("slow")
+    def download_unathorized_test(self):
+        http_server = StoppableThreadBottle()
+
+        @http_server.server.get('/forbidden')
+        def get_forbidden():
+            return HTTPError(403, "Access denied.")
+
+        http_server.run_server()
+
+        out = TestBufferConanOutput()
+        dest = os.path.join(temp_folder(), "manual.html")
+        # Not authorized
+        with self.assertRaises(ConanException):
```

Comment:
Check that the exception has something to do with the 403, it might be Conan failing somewhere else